File: LeetCode/Simulation/Python/Leetcode1684-count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        ans, mask = 0, 0
        for c in allowed:
            mask |= 1 << (ord(c) - ord('a'))
        for word in words:
            cur = 0
            for c in word:
                cur |= 1 << (ord(c) - ord('a'))
            ans += (mask | cur) == mask
        return ans

File: LeetCode/Simulation/JS/Leetcode1684-count-the-number-of-consistent-strings.js
var countConsistentStrings = function(allowed, words) {
    let [ans, mask] = [0, 0];
    for(const c of allowed) mask |= 1 << (c.charCodeAt() - 'a'.charCodeAt());
    for(const word of words) {
        let cur = 0;
        for(const c of word) {
            cur |= 1 << (c.charCodeAt() - 'a'.charCodeAt());
        } 
        ans += (mask | cur) == mask;
    }
    return ans;
};
